File: prime.js
let number=2;
function isPrime(number)
{
    for(let i=2;parseInt(i)<=Math.sqrt(number);i++)
{
     if(number%i==0)
    return "not prime";
    break;
}
    return "prime";
}
console.log(isPrime(number));

File: Lec24/Prime.java
package Lec24;

public class Prime {

    public static void main(String[] args) {
        int n=10;
        int i=2;
        int count=0;
        while(count<n){
            if(IsPrime(i)){
                System.out.println(i);
                count++;
            }
            i++;
        }
    }
    public static boolean IsPrime(int n){
        for(int i=2;i<=Math.sqrt(n);i++){
            if(n%i==0){
                return false;
            }
        }
        return true;
    }
}
